docs/modules/automation-testing/pages/scripting/create-image-injection-script.adoc

```diff
@@ -0,0 +1,98 @@
+= Create image injection script
+:navtitle: Create an image injection script
+
+Learn how to automate the process of image injection, so you can automate your test suite.
+
+== Before you start
+
+Enable image injection for your app with either of these options:
+
+* Either use the capability `'kobiton:instrumentenabled'` to xref:#appium_instrumentation_capability[instrument] your app.
+

```

Comment:
The first sentence says "either of these options" so the Either and Or are not needed at the beginning of the bullets and are not a standard format. 
Suggestion:
In first sentence - change "either" to "one"
In first bullet - remove Either and change to "Instrument your app and use the capability 'kobiton:instrumentenabled' (Android only)" with Instrument your app linked
In second bullet: remove "Or" and start with "Add..."
Also in second bullet - Android is misspelled and should be capitalized

Reply:
Changed according to suggestion.